scripts/util.py

```diff
@@ -152,6 +155,61 @@
     None
 ] = lambda self, data: yaml.representer.SafeRepresenter.represent_str(self, str(data))
 
+with open(PROVIDERS_FILE) as stream:
+    try:
+        providers = yaml.safe_load(stream)
+    except yaml.YAMLError as exc:
+        providers = None
+        print(exc)
+prv = NSDict(providers)
```

Comment:
We should avoid using NSDict, it's too promiscuous and can lead to bugs.

Reply:
Obsolete.